Paccy10/olympus-api
# Feature(properties): create new property

**What does this PR do?**

- Create new property

**Description of Task to be completed?**

- An authenticated user can be able to post a new property

**How should this be manually tested?**

- Checkout to the branch `ft-create-new-property`
- Run the app with `flask run`
- Go to `postman` and send a `POST` request to /`properties` with the required payload
- A new property should be added to the database

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
